### PR TITLE
Fix broken links aqua-identity-protocol.md

### PR DESCRIPTION
**Pull request by girijakar**

### CODE REVIEW

File: content/en/docs/protocol/aqua-identity-protocol.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Self-Sovereign-Identity provided by Christopher Allen in shortened form here.[^4
 
 Those 10 stated principles are implemented within the Aqua Protocol and
 it's existing reference implementation in various degrees. They are also
-represented in the [Design Principles](/docs/protocol/design-principles/).
+represented in the [Design Principles](design-principles.md).
```
Comment (rht) [inline content/en/docs/protocol/aqua-identity-protocol.md:39]: Not sure about this one. If I check this link in https://aqua-protocol.org/docs/protocol/aqua-identity-protocol/, the current main already works.

But we might want to just say design-principles.md to make it simpler. I'm not sure if not specifying the absolute path will work.

It's also not good either to impose absolute pathing that may change depending on how aqua-docs is hosted.

cc: @ux-design-dev

Comment (girijakar) [reply content/en/docs/protocol/aqua-identity-protocol.md:39]: The link is also working for design principles. But if you will check from the markdown, it shows 404 error.

Comment (girijakar) [reply content/en/docs/protocol/aqua-identity-protocol.md:39]: I've found some typos & spelling mistakes. I would like to address those also, if you are fine with it I will work on those after fixing the broken links issue.

Comment (rht) [reply content/en/docs/protocol/aqua-identity-protocol.md:39]: @girijakar  sounds good for typos & spelling mistakes. There are 2 tools you can use for this: [Codespell](https://github.com/codespell-project/codespell) and [LanguageTool](https://github.com/languagetool-org/languagetool).
Codespell only checks for typos, but LanguageTool is more comprehensive and checks for grammar problems.

Ideally, we should at least automate the Codespell checking in the CI.
You can see how it is done in https://github.com/inblockio/PKC-Content/blob/e7efc4170c3c0fda220592b78a53bea270b88077/.github/workflows/ci.yml#L9-L17.

Comment (rht) [reply content/en/docs/protocol/aqua-identity-protocol.md:39]: I raised #12 for Codespell.

**Comment by rht** [comment]
Merged, thank you!

**Pull request merged**